File: backend/src/main/java/com/back/domain/review/review/dto/ReviewResponseDto.java
package com.back.domain.review.review.dto;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDateTime;

@Getter
@Builder
public class ReviewResponseDto {
    private int id;
    private String content;
    private int rate;
    private String memberName;
    private int memberId;
    private int likeCount;
    private int dislikeCount;
    private Boolean isRecommended;
    private LocalDateTime createdDate;
    private LocalDateTime modifiedDate;
}

File: backend/src/main/java/com/back/domain/review/review/dto/ReviewResponseDto.kt
package com.back.domain.review.review.dto

import com.back.domain.review.reviewReport.entity.ReviewReportState
import java.time.LocalDateTime


data class ReviewResponseDto(
    val id: Int,
    val content: String,
    val rate: Int,
    val memberName: String,
    val memberId: Int,
    val likeCount: Int,
    val dislikeCount: Int,
    val spoiler: Boolean,
    val isRecommended: Boolean?,
    val reportState: ReviewReportState,
    val adminMessage: String?,
    val createdDate: LocalDateTime,
    val modifiedDate: LocalDateTime
) {
}
